target/linux/realtek/dts/rtl9312_plasmacloud_psx28.dts

```diff
@@ -0,0 +1,700 @@
+			sds = < 4 >;
+		};
+		phy9: ethernet-phy@17 {
+			reg = <17>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 9>;
+			sds = < 4 >;
+		};
+		phy10: ethernet-phy@20 {
+			reg = <20>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 10>;
+			sds = < 4 >;
+		};
+		phy11: ethernet-phy@21 {
+			reg = <21>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 11>;
+			sds = < 4 >;
+		};
+		phy12: ethernet-phy@24 {
+			reg = <24>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 12>;
+			sds = < 5 >;
+		};
+		phy13: ethernet-phy@25 {
+			reg = <25>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 13>;
+			sds = < 5 >;
+		};
+		phy14: ethernet-phy@28 {
+			reg = <28>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 14>;
+			sds = < 5 >;
+		};
+		phy15: ethernet-phy@29 {
+			reg = <29>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 15>;
+			sds = < 5 >;
+		};
+		phy16: ethernet-phy@32 {
+			reg = <32>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 16>;
+			sds = < 6 >;
+		};
+		phy17: ethernet-phy@33 {
+			reg = <33>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 17>;
+			sds = < 6 >;
+		};
+		phy18: ethernet-phy@36 {
+			reg = <36>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 18>;
+			sds = < 6 >;
+		};
+		phy19: ethernet-phy@37 {
+			reg = <37>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 19>;
+			sds = < 6 >;
+		};
+		phy20: ethernet-phy@40 {
+			reg = <40>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 20>;
+			sds = < 7 >;
+		};
+		phy21: ethernet-phy@41 {
+			reg = <41>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 21>;
+			sds = < 7 >;
+		};
+		phy22: ethernet-phy@44 {
+			reg = <44>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 22>;
+			sds = < 7 >;
+		};
+		phy23: ethernet-phy@45 {
+			reg = <45>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <1 23>;
+			sds = < 7 >;
+		};
+
+		/* fiber */
+	#if defined(ENABLE_SFP_PHYS)
+		phy24: ethernet-phy@48 {
+			compatible = "ethernet-phy-ieee802.3-c45";
+			phy-is-integrated;
+			reg = <48>;
+			rtl9300,smi-address = <0 0>;
+			sds = <8>;
+		};
+
+		phy25: ethernet-phy@50 {
+			compatible = "ethernet-phy-ieee802.3-c45";
+			phy-is-integrated;
+			reg = <50>;
+			rtl9300,smi-addresss = <0 0>;
+			sds = <9>;
+		};
+
+		phy26: ethernet-phy@52 {
+			compatible = "ethernet-phy-ieee802.3-c45";
+			phy-is-integrated;
+			reg = <52>;
+			rtl9300,smi-address = <0 0>;
+			sds = <10>;
+		};
+
+		phy27: ethernet-phy@53 {
+			compatible = "ethernet-phy-ieee802.3-c45";
+			phy-is-integrated;
+			reg = <53>;
+			rtl9300,smi-address = <0 0>;
+			sds = <11>;
+		};
```

Comment:
Please check if you really need to use clause 45 for the integrated phy of the SerDes.

Usually, I would expected to see following block:


```suggestion
		INTERNAL_PHY_SDS(48, 8)
		INTERNAL_PHY_SDS(50, 9)
		INTERNAL_PHY_SDS(52, 10)
		INTERNAL_PHY_SDS(53, 11)
```

But this would then configure it as `ethernet-phy-ieee802.3-c22` - which all other devices use for the "same" SerDes.